asv_bench/benchmarks/array.py

```diff
@@ -80,7 +78,7 @@ def setup(self, multiple_chunks):
             import pyarrow as pa
         except ImportError:
             raise NotImplementedError
-        strings = tm.rands_array(3, 10_000)
+        strings = np.array(["a"] * 10_000, dtype=object)
```

Comment:
there's no scenario where having random data could make a difference?  i could imagine pyarrow having some optimization for constant data

Reply:
Not sure if they hash values or anything but that's a good point. I'll make sure the benchmark use unique values